graphql-schemas/schema.yml

```diff
@@ -9,8 +9,8 @@
   - { name: kubeBinary, type: string, isRequired: true }
   - { name: pullRequestGateway, type: string }
   - { name: mergeRequestGateway, type: string }
-  - { name: saasDeployJobTemplate, type: string }
-  - { name: hashLength, type: int }
+  - { name: saasDeployJobTemplate, type: string, isRequired: true }
```

Comment:
We could also loose these kinds of restrictions

You can simply remove all the changes in graphql-schemas/schema.yml that only add `isRequired: true` for the existing field.